### ISSUE TITLE
Utilise HPC for Data Generation Scripts

### ISSUE THREAD

**Issue by grayJiaaoLi** [amosproj/amos2024ss08-cloud-native-llm#71]
## User story
1. As a data engineer
2. I want/need to run data generation scripts on HPC
3. So that we can have the Q&A pairs training data ready for model training and fine-tuning

## Acceptance criteria
* Run the scripts for the Q&A pairs data generation on HPC

* Ensure the output data is ready for further LLM training and fine-tuning

* Store the Q&A pairs on HuggingFace

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by dominic0df** [comment]
added DoD
